src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -872,6 +872,7 @@ public override void VisitParameterReference(IParameterReferenceOperation operat
         public override void VisitInstanceReference(IInstanceReferenceOperation operation)
```

Comment:
>VisitInstanceReference [](start = 29, length = 22)

Consider making an adjustment to TestOperationVisitor as well. At least we should verify that the new property doesn't throw.